docs/shared/migration/from-turborepo.md

```diff
@@ -0,0 +1,189 @@
+| `--dry-run`                 | Same syntax                                                                                                                                                                             |
+| `--env-mode`                | N/A                                                                                                                                                                                     |
+| `--filter`                  | Use [`-p admin-*` or `-p tag:api-*`](/packages/nx/documents/run-many#projects). Also see [`nx affected`](/packages/nx/documents/affected).                                              |
+| `--graph`                   | [Same syntax](/packages/nx/documents/run-many#graph) or [`nx graph`](/packages/nx/documents/dep-graph) for the entire graph                                                             |
+| `--force`                   | [`nx reset`](/packages/nx/documents/reset) and then run the command again                                                                                                               |
+| `--global-deps`             | Use [`inputs` in the `nx.json`](/concepts/more-concepts/customizing-inputs) or project configuration                                                                                    |
+| `--framework-inference`     | Nx knows if you're using a particular framework if you use an executor for that framework.                                                                                              |
+| `--ignore`                  | Use an [`.nxignore` file](/reference/nxignore) (or `.gitignore`)                                                                                                                        |
+| `--log-order`               | Use [`--output-style`](/packages/nx/documents/run-many#output-style)                                                                                                                    |
+| `--no-cache`                | Use [`--skip-nx-cache`](/packages/nx/documents/run-many#skip-nx-cache)                                                                                                                  |
+| `--no-daemon`               | Use [`NX_DAEMON=false` or set `useDaemonProcess: false`](/concepts/more-concepts/nx-daemon#turning-it-off) in `nx.json`                                                                 |
+| `--output-logs`             | Use [`--output-style`](/packages/nx/documents/run-many#output-style)                                                                                                                    |
+| `--only`                    | N/A. Could [use `--exclude`](/packages/nx/documents/run-many#exclude) to ignore certain projects.                                                                                       |
```

Comment:
exclude is different, so I wouldn't even mention it. They exclude dependent targets to be run with `--only` (which is weird). `exclude` in our case excludes projects